wire-runtime/src/main/java/com/squareup/wire/ExtensionRegistry.java

```diff
@@ -15,43 +15,29 @@
  */
 package com.squareup.wire;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.LinkedHashMap;
+import java.util.List;
 import java.util.Map;
 
 final class ExtensionRegistry {
```

Comment:
We should probably just fold this type into `Wire` as both are now tiny versions of their previous selves. Will do in a follow-up.